### PR TITLE
Playlist lists

### PR DESCRIPTION
**Pull request by NissesSenap**

### CODE REVIEW

**Review by simongottschlag — COMMENTED** [review]
Left two comments

File: playlist.go [hunk 1]
```diff
@@ -44,6 +50,19 @@ func (c *Client) Playlist(idOrUID string) (*Playlist, error) {
 	return playlist, nil
 }
 
+// Playlists fetches and returns a Grafana playlist.
+func (c *Client) Playlists(params url.Values) (*Playlists, error) {
```
[the remark this answers is not in the document]
Comment (NissesSenap) [reply playlist.go:54]: It's a good point but I think I will keep it like this mostly due to how they have defined it earlier.
https://github.com/NissesSenap/grafana-api-golang-client/blob/8d9d243998b12e8d9b867c6ad181f9c44aebf828/folder_dashboard_search.go#L26

I have created the upstream PR now so lets see what they say.

[the remark this answers is not in the document]
Comment (simongottschlag) [reply playlist.go:54]: The problem with using `url.Values` is that you will need to read other code to understand how you should create it outside of the function. It won't be obvious that `query` and `limit` are the values that can be used and what their characteristics are.

File: playlist_test.go [hunk 2]
```diff
@@ -91,6 +94,52 @@ func TestGetPlaylist(t *testing.T) {
...
+		t.Errorf("Unable to unmarshal getPlaylistResponse - %s", getPlaylistResponse)
+	}
+
+	playlists, err := client.Playlists(url.Values{})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	for _, playlist := range playlists.Playlist {
+		if playlist.UID == "2" {
```
[the remark this answers is not in the document]
Comment (NissesSenap) [reply playlist_test.go:113]: Agreed.

**Comment by NissesSenap** [comment]
I have created a upstream PR now: https://github.com/grafana/grafana-api-golang-client/pull/113

**Pull request closed without merging**